Added twitter to the testimonials section

sumitkr2000 opened the pull request:
<!-- Pull Request Template -->

## Related Issue

Closes #1175 

<!-- If there is no issue number, the PR will not be merged. Therefore, please ensure that the issue number is added -->

## Description

Hey @rohansx I have added Twitter URLs to the testimonials section so that users can see the details of the admins and mentors.

## Screenshots

![image](https://github.com/rohansx/informatician/assets/100675296/cd7acef4-94cf-48ee-824c-a57a08bb86e3)

## Checklist 

<!-- [x] - To mark checked, put 'x' in place of ' '(space)  -->
<!-- [ ] - Keep unchecked using ' '(space)  -->

- [x] My code adheres to the established style guidelines of the project.
- [x] I have included comments in areas that may be difficult to understand.
- [x] My changes have not introduced any new warnings.
- [x] I have conducted a self-review of my code.

sumitkr2000 commented:
> Add twitter link & github link also

okay

sumitkr2000 commented:
@rohansx done

AlfiyaSiddique commented:
@rohansx This PR issue is similar to my PR issue #1187 so can you please look into this matter before it create problems.

sumitkr2000 commented:
> @rohansx This PR issue is similar to my PR issue #1187 so can you please look into this matter before it create problems.

okay sure I have added all the three links

SiddhantPawar03 commented:
@sumitkr2000 please look into it as this issue has already been solved.

sumitkr2000 commented:
> @sumitkr2000 please look into it as this issue has already been solved.

i have added the twitter link also

sumitkr2000 commented:
![image](https://github.com/rohansx/informatician/assets/100675296/fa0ef06d-a565-418d-a70b-e3b5a662f8fb)

SiddhantPawar03 commented:
> > @sumitkr2000 please look into it as this issue has already been solved.
> 
> i have added the twitter link also

Actually #1136 is the similar issue and your's was assigned later. Still can approve for twitter links!
Requesting to change title.

sumitkr2000 commented:
> > > @sumitkr2000 please look into it as this issue has already been solved.
> > 
> > 
> > i have added the twitter link also
> 
> Actually #1136 is the similar issue and your's was assigned later. Still can approve for twitter links! Requesting to change title.

done ... i have changed the title
